masterlist.yaml

```
@@ -4882,6 +4882,24 @@ plugins:
         itm: 27
         udr: 8
 
+  - name: 'DEJ_realSize.*\.esp'
```

Since there are only 3 plugins & the mod hasn't been updated for several years, the regex can be made less greedy:

```suggestion
  - name: 'DEJ_realSize(City|Dungeon|Wildreness)Rats\.esp'
```